iati/core/tests/test_default.py

```diff
@@ -48,6 +48,40 @@ def test_default_codelists(self):
         for _, codelist in codelists.items():
             assert isinstance(codelist, iati.core.Codelist)
 
+    def test_codelist_mapping_condition(self):
+        """Check that the Codelist mapping file is being read correctly.
+
+        Todo:
+            Split into multiple tests.
+        """
+        mapping = iati.core.default.codelist_mapping()
+
+        assert mapping['Sector'][0]['condition'] == "@vocabulary = '1' or not(@vocabulary)"
+        assert mapping['Version'][0]['condition'] is None
+
+    def test_codelist_mapping_xpath(self):
+        """Check that the Codelist mapping file is being read correctly.
+
+        Todo:
+            Split into multiple tests.
+        """
+        mapping = iati.core.default.codelist_mapping()
+
+        assert mapping['Version'][0]['xpath'] == '//iati-activities/@version'
```

Comment:
Would also be good to test for codelists which have more than one mapping. In the case of the `Version` codelist, this maps to `//iati-activities/@version` and `//iati-organisations/@version`. The `Sector` codelist is another with multiple mappings.

Reply:
Relates to the validation tests for the `valid_iati_invalid_codes_multiple_xpaths_for_codelist_[first|second].xml` files

Reply:
https://github.com/IATI/iati.core/pull/142/commits/268a9b5292364ac88c6ae6547344a97799803aa7